VERSION ""

NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    BA_DEF_DEF_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_SGTYPE_
    BA_SGTYPE_
    SIG_TYPE_REF_
    VAL_TABLE_
    SIG_GROUP_
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    BO_TX_BU_
    BA_DEF_REL_
    BA_REL_
    BA_DEF_DEF_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_:
BU_: RaspPi1 RaspPi2

BO_ 300 SensorData: 8 RaspPi1
 SG_ temperature : 0|8@1+ (1,0) [0|255] "Â°C" RaspPi2
 SG_ voltage     : 8|16@1+ (0.1,0) [0|6553.5] "V" RaspPi2
 SG_ current     : 24|12@1+ (0.01,0) [0|40.95] "A" RaspPi2
 SG_ status_flag : 36|1@1+ (1,0) [0|1] "" RaspPi2

BO_ 301 AckMessage: 1 RaspPi2
 SG_ ack : 0|1@1+ (1,0) [0|1] "" RaspPi1
